cargo-near/src/abi.rs

```diff
@@ -54,7 +52,9 @@ pub(crate) fn write_to_file(
         strip_docs(&mut contract_abi);
     }
     let near_abi_json = serde_json::to_string(&contract_abi)?;
-    let out_path_abi = crate_metadata.target_directory.join(ABI_FILE);
+    let out_path_abi = crate_metadata
+        .target_directory
+        .join(crate_metadata.root_package.name.replace('-', "_") + "_abi.json");
```

Comment:
This PR also prepends the package name to the name of the ABI file.

If you'd prefer to leave the ABI file as `abi.json`, there's another path I can take to arrive at the same result without changing the filename.

lmk what you think.

Reply:
I think this is fine, no need for #42